docs/0.4/concepts/biome_user_management.md

```diff
@@ -46,7 +46,7 @@ exposes Biome functionality as Biome-specific endpoints. The following diagram
 shows the relationship of the Biome component and REST API routes with the other
 `splinterd` components and routes.
 
-<img alt="Biome architecture" src="../images/biome_architecture_diagram.svg">
+<img alt="Biome architecture" src="/docs/images/biome_architecture_diagram.svg">
```

Comment:
These images should be versioned 0.4/0.5 because we might change them separately.

Reply:
@vaporos - Instead of creating release-specific versions of each image, what about moving the `docs/images` directory to `docs/0.4/images/` and `docs/0.5/images/`?  That way, we won't have to update the image links each time we have a new release version.

Reply:
P.S.  I suspect that's what you meant in your original comment. (The `/docs0.4/images/` and `/docs/0.5/images` folders already exist.) If so, I can push a commit that moves the Biome images to those versioned folders.

Reply:
Moved the images to the right "versioned" subdirectory.